openapi/SwarmCommon.yaml

```diff
@@ -385,6 +385,8 @@ components:
       properties:
         batchID:
           $ref: "#/components/schemas/BatchID"
+        txHash:
```

Comment:
please update version

Reply:
@istae sorry, which version should it be?